modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1757,6 +1758,11 @@ export class Wallet implements IWallet {
    */
   async send(params: SendOptions = {}): Promise<any> {
     common.validateParams(params, ['address'], ['message', 'data']);
+    // Note: address and amount not required for enable token
+    if (params.type == 'enabletoken') {
+      params.address = params.address || 'not required';
+      params.amount = params.amount || 1;
+    }
```

Comment:
sorry to flip flop, but lets switch back and NOT set default values for `address` and `amount`.

```
if (params.type !== 'enabletoken') {
   // do validation
}
```

Reply:
Ohh okk suree, yes default value + set to empty is kinda ugly in terms of design so yeahh will change to that